clang-tools-extra/unittests/clang-tidy/CMakeLists.txt

```diff
@@ -39,6 +40,7 @@ add_extra_unittest(ClangTidyTests
 
 clang_target_link_libraries(ClangTidyTests
   PRIVATE
+  clangAnalysis
```

Comment:
Why is this needed? Is it the same as the Clang Static Analyzer?

Reply:
This is for `clang::CFG`. The `#include` was missing, thanks for the catch.